package.json

```diff
@@ -44,6 +44,7 @@
     "webpack-dev-server": "^4.9.0"
   },
   "dependencies": {
+    "i18next": "^21.8.6",
```

Comment:
we should be using exact versions instead of fuzzy versions in package.json (same for the other deps)